### ISSUE TITLE
[SF][AnalyticalTable]: Row selection checkboxes missing aria-hidden="true"

### ISSUE THREAD

**Issue by summer2200** [SAP/ui5-webcomponents-react#6133]
### Describe the bug

From this issue https://github.com/SAP/ui5-webcomponents-react/issues/5075, we got to know that the checkboxes are not supposed to be focused, but they can still be detected by axe tool (a tool which used by A11y team to check related issues).
 
Please add **aria-hidden="true"** to the checkboxes so that they will not be scanned by axe tool.
![Screenshot 2024-07-29 at 16 47 47](https://github.com/user-attachments/assets/69f8b24e-2fdc-45b7-adea-789ea9926a9f)

### Reproduction steps

1. Add "axe DevTools" extension in chrome
2. Login to system
3. Scan the page with axe tool
...

### Expected Behaviour

Checkboxes should have aria-hidden attribute as true.

### UI5 Web Components for React Version

1.29.4

### UI5 Web Components Version

1.24.7

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.
